src/style.css

```
@@ -0,0 +1,75 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  background-color: black;
+  --blue-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.image {
+  height: 40px;
+  margin-top: 10px;
```

This image is situated inside the flex-container. You could easily center it vertically with 
```suggestion
  align-self: center;
```

---

OK, thanks!